### DISCUSSION TITLE
Problem Tasmota ESP32-C3 EU Plug V3 network scan restart

### DISCUSSION THREAD

**Discussion by saviothecnic** [arendst/Tasmota#20870] [Support request]
Guys I have some Athom sockets with Tasmota 32 on
I contacted the supplier and he told me to hear directly
Tasmota support
I realized that if I scan the rere
type with Advanced IP Scanner program
when the program scans IP of the tasmota socket
this restarts by disconnecting the load applied to it
and this is very annoying because I have these sockets even on computers that are taken while scanning the network
this does not happen with sockets that have normal tasmota
on ESP8266 but only on sockets with ESP32
I also added a password to access Tasmnota
but this didn't solve the problem

**Comment by arendst** [comment]
Pls report output of command ``status 0``

**Comment by arendst** [comment]
Try an update to latest release and report back.

**Comment by arendst** [comment]
Verified that Advanced IP Scanner does break the Athom V3 (ESP32-C3) and Shelly Plus 1PM (ESP32-C3) power plug (?). I have three Athoms and 1 Shelly. During the scan all four restarted with an exception 7. Other 38 Tasmota devices (all non-esp32-c3) did not restart.

I'll dive in to see if I can find the cause and possibly a solution. In the meantime stop using the Advanced IP scanner which chrome also finds  flaky considering the fact that the download of the program needs my acceptance before use AND that it seems to do a lot more than just polling for open ports shown by multiple Open Main Menu messages...

**Reply by saviothecnic** [reply]
Ok thanks for your quick check
Yes, the tools, in addition to scanning the network, checks whether there are services and ports open and the user connected to the IP
It does a very thorough scan on your devices
I use it a lot for work
So I would be happy if you can solve the anomaly. Thanks

**Comment by arendst** [comment]
Update. It seems to fail ONLY on risc based hardware (ESP32-C3) and fails in the SDK so out of Tasmota's control.

I currently chk if core 3 (soon to be released) also has this problem...

**Comment by arendst** [comment]
You're in luck!! Core 3 doesn't restart ESP32-C3 devices when scanned by Advanced IP Scanner.

So hold your breath until Arduino Core 3 officially is released. We then release Tasmota v14 which fixes your issue.

**Reply by saviothecnic** [reply]
Yes I thought that by putting the password
the device instead resolved no
Obviously for now I won't use the scanning tools
But I think this is also a security issue
If someone with bad feelings achieves something
looping these requests over the network makes Tasmota devices based on this hardware unusable and that's not nice

**Reply by arendst** [reply]
As a workaround disable scanning for HTTP in the IP scanner. This solves the restarts with exception 7 too.

**Reply by sfromis** [reply]
Tasmota is not trying to be safe against attackers you've provided network access.

If you want to use core 3, you could do a custom build.

**Reply by saviothecnic** [reply]
I understand I have these sockets
I'm not good at being able to set up a custom thing
I hope my report can help
in a future upgrade to fix the problem

**Reply by saviothecnic** [reply]
News you managed to solve the problem

**Comment by saviothecnic** [comment]
In the meantime, I confirm that by disabling HTTP
now scan the network without rebooting your device
